lib/myelements/add_objects.py

```diff
@@ -58,7 +58,8 @@ def triangle(
             restitution=0.16,
             friction=0.5,
             screenCoord=True):
-        """ Add a triangle | pos & a in the current input unit system (meters or pixels)
+        """ Add a triangle | pos & a in the current input unit system
+            (meters or pixels)
```

Comment:
This would be better if the quotes are on its own line, same with every other one like it.

Reply:
Sure, fixed